src/main/java/net/earthcomputer/multiconnect/protocols/v1_9_4/mixin/MixinLivingEntity.java

```diff
@@ -0,0 +1,21 @@
+package net.earthcomputer.multiconnect.protocols.v1_9_4.mixin;
+
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(LivingEntity.class)
+public class MixinLivingEntity {
+
+    @Inject(method = "getPreferredEquipmentSlot", at = @At("HEAD"), cancellable = true)
+    private static void removeShieldSlotPreference(ItemStack stack, CallbackInfoReturnable<EquipmentSlot> cir) {
+        if(stack.isOf(Items.SHIELD))
```

Comment:
Need to check the protocol version. Also can you put braces on the if statement?